tools/modules/EnsEMBL/Web/Component/Tools/VEP/Results.pm

```diff
@@ -145,6 +145,15 @@ sub content {
   my $headers = $header_hash->{'combined'};
   my $header_extra_descriptions = $header_hash->{'descriptions'} || {};
 
+  # Overwrite DisGeNET header description
+  # Description example: "Variant-Disease-PMID associations from the DisGeNET database. The output includes 
+  #   the PMID of the publication reporting the Variant-Disease association, DisGeNET score for the Variant-Disease association, 
+  #   name of associated disease. Each value is separated by ':'"
+  # In Web VEP, the values are not separated by ':'
+  if(exists $header_extra_descriptions->{'DisGeNET'}) {
+    $header_extra_descriptions->{'DisGeNET'} =~ s/Each value is separated.*//;
```

Comment:
Thank you for the description example. There is an extra space which you also need to match in your regex before **Each ...**.  And just to clarify the the comment I would add: In Web VEP, the values are not separated by ':' **and therefore remove it from the description.**